Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValue.cs

```diff
@@ -654,6 +669,8 @@ protected bool IsSelectStatementEnabled
 
     void IBocReferenceValue.PopulateDropDownList (DropDownList dropDownList)
     {
+      // PopulateDropDownList should be moved to the renderer, the BocReferenceValue should only provide a list of items, see also BocEnumValue.
+
```

Comment:
Wäre das nicht besser als Task?

Reply:
In theory. Thing is, I don't think this will ever get scheduled unless I happen to stumble over it during a refactoring that touches this part of the code. It's a) more work than it seems so there won't be time to just-do-it and b) completely workable at this point. So, making a task for it right now will just add a couple of random bytes to the backlog and when it comes around, you won't know that there's a task for it. and adding the task and a todo comment just felt like too much hassle at this point.